tests/test_training.py

```diff
@@ -52,6 +55,30 @@ def create_c4_dataset_xsmall(prefix: str) -> str:
     return c4_dir
 
 
+def create_arxiv_dataset(prefix: str) -> str:
+    """Creates an arxiv dataset."""
+    arxiv_dir = os.path.join(os.getcwd(), f'my-copy-arxiv-{prefix}')
+    shutil.rmtree(arxiv_dir, ignore_errors=True)
```

Comment:
Let's use `tmp_path` here. We can pass it as an argument to this function, and we should make arxiv_dir live in `tmp_path` not `os.getcwd()`. That way, we can also remove L61.

The above function should use tmp_path too, so let's just make the change for the whole file?

(`tmp_path` is a pytest fixture, and automatically gets cleaned up, so it's nice to use here so we don't have tests that create folders and that worry about removing them.)

Reply:
done!